### PR TITLE
add istanbul tool hint

### PR DESCRIPTION
**Pull request by magemello**

### CODE REVIEW

**Comment by stevepeak** [comment]
@eddiemoore thoughts? Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ install:
   - npm install --dev
 script:
   - npm test
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (eddiemoore) [inline .travis.yml]: This is already handled by the `posttest` script.

**Comment by eddiemoore** [comment]
@magemello If showing how to end a test with remap-istanbul, probably best to actually show how to test with istanbul too, as some people may not know how to do that.

**Comment by thomasrockhu** [comment]
Hi @magemello, thanks for the work on this PR.  Due to the age of the request however, I'm going to close it out.  Feel free to re-open if you feel it would be useful.

**Pull request closed without merging**